src/debug/debug_impl.ts

```diff
@@ -146,6 +146,20 @@ export class DartDebugSession extends DebugSession {
 							let thread = this.threadManager.getThreadInfoFromRef(isolateRef);
 							thread.receivedPauseStart();
 						}
+
+						// Helpers to categories libraries as SDK/ExternalLibrary/not.
+						let localPackageName = this.packageMap.getLocalPackageName();
+						let isSdkLibrary = (l: VMLibraryRef) => l.name.startsWith("dart.") || l.name == "builtin";
+						let isExternalLibrary = (l: VMLibraryRef) => !isSdkLibrary(l) && l.uri.startsWith("package:") && !l.uri.startsWith(`package:${localPackageName}/`);
```

Comment:
> I think you can remove the initial `!isSdkLibrary(l) &&` check

I managed to do this - I change the `isSdk` function to check the `uri` for `dart:`. This seems more reliable anyway (I don't know if anything stops people from using `dart.` in their own library names, and I don't need to special-case `builtin` then).